console/console-init/ui/src/modules/iot-device/containers/DeviceListContainer/DeviceListContainer.tsx

```diff
@@ -159,46 +152,21 @@ export const DeviceListContainer: React.FC<IDeviceListContainerProps> = ({
     });
   };
 
-  const disableDevice = (row: any) => {
-    const { deviceId, jsonData, viaGateway } = row.originalData;
-    const device = {
-      deviceId,
-      jsonData,
-      viaGateway,
-      enabled: false
-    };
+  const toggleDeviceStatus = (row: any, status: boolean) => {
```

Comment:
If we are referring to table row here, type of table row could have been used instead of any.